fsh/terminology/regimenType.fsh

```diff
@@ -8,4 +8,118 @@ Description: "The type of regimen - eg FOLFOX"
 * ^url = "https://standards.digital.health.nz/fhir/ValueSet/sact-regimen-code"
 * ^jurisdiction.coding = urn:iso:std:iso:3166#NZ
 
-* $SNOMED#dummy "FOLFOX"
+* $SNOMED#ABVD "ABVD"
```

Comment:
As you will see this valueset isn't snomed codes.  I have left it in the same format but obviously that will need to change.  I know we discussed this but I couldn't remember what to do to specify this